public/app/features/explore/Wrapper.tsx

```diff
@@ -86,22 +118,56 @@ class WrapperUnconnected extends PureComponent<Props> {
     document.title = documentTitle;
   }
 
+  updateSplitSize(rightPaneWidth: number) {
+    const evenSplitWidth = window.innerWidth / 2;
+    const areBothSimilar = inRange(rightPaneWidth, evenSplitWidth - 100, evenSplitWidth + 100);
+    if (areBothSimilar) {
+      this.props.changeLargerPane(undefined);
+    } else {
+      this.props.changeLargerPane(rightPaneWidth > evenSplitWidth ? ExploreId.right : ExploreId.left);
+    }
+
+    this.setState({ rightPaneWidth });
```

Comment:
We did end up needing to keep the width in component state here. This was because a manual resize would look fine fine, but the `widthCalc` would remain constant, so trying to do a minimize/maximize after a manual resize wouldn't work. We need the widthCalc to reflect every width change.

To see what I'm talking about, try a commit before this one. Manually drag the bar to one side, and hit - on the larger pane. It won't work until you click it twice more.